### PR TITLE
feat(interviews): style interviews-card

### PR DESCRIPTION
**Pull request by tipsy**

### CODE REVIEW

File: src/components/interview-information/interview-information.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+
+  &__what {
+    text-align: right;
+  }
+
+  &__time {
+    color: #555;
```
Comment (pkkummermo) [inline src/components/interview-information/interview-information.scss]: Separate colors into variables on top?

File: src/components/interview-information/interview-information.ts [hunk 2]
```diff
@@ -4,14 +4,19 @@ import Vue from "vue";
...
 @Component({
     template: require("./interview-information.html"),
 })
 export class InterviewOverviewComponent extends Vue {
     interviewService = new InterviewService();
     interview: Interview = null;
-    formatMoment(starttime: string | moment.Moment, endtime: string | moment.Moment) {
-        return moment(starttime).format("HH:mm") + " - " + moment(endtime).format("HH:mm (DD. MMM - YYYY)");
+    formatHours(starttime: string | moment.Moment, endtime: string | moment.Moment) {
+        return moment(starttime).format("HH:mm") + " - " + moment(endtime).format("HH:mm");
```
Comment (pkkummermo) [inline src/components/interview-information/interview-information.ts]: Use string interpolation! The future is here :D

File: src/components/interview-information/interview-information.scss (hunk 1 above)
Comment (tipsy) [reply src/components/interview-information/interview-information.scss]: Done

File: src/components/interview-information/interview-information.ts (hunk 2 above)
Comment (tipsy) [reply src/components/interview-information/interview-information.ts]: Done

**Pull request merged**